components/admin_console/permission_schemes_settings/index.tsx

```diff
@@ -21,9 +25,9 @@ function mapStateToProps(state) {
     };
 }
 
-function mapDispatchToProps(dispatch) {
+function mapDispatchToProps(dispatch: Dispatch<GenericAction>) {
     return {
-        actions: bindActionCreators({
+        actions: bindActionCreators<ActionCreatorsMapObject<ActionFunc| GenericAction>, Props['actions']>({
```

Comment:
just for consistent spaces, but not a big deal 0/5
```suggestion
        actions: bindActionCreators<ActionCreatorsMapObject<ActionFunc | GenericAction>, Props['actions']>({
```

Reply:
fixed on 6f2facaf2fcb999741899fa9e461cf4a4cd5bcf7